Test/comp/firstSteps/7_Arrays.dfy.expect

```diff
@@ -0,0 +1,14 @@
+
+Dafny program verifier finished with 7 verified, 0 errors
+
+Dafny program verifier did not attempt verification
+0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 
+17
+[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]
+[2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
+[20, 21, 22]
+[0, 1, 2, 3, 4, 5, 6, 7]
+[0, 1, 2, 3, 4, 5, 6, 7]
+0 0 0 0 0 0 0 0 
+0 0 0 0 0 0 0 0 
+[] [0] [0, 0] [0, 0, 0] [0, 0, 0, 0]
```

Comment:
```suggestion
[] [0] [0, 0] [0, 0, 0] [0, 0, 0, 0] 
```

Reply:
Can I know , why are you suggesting this ? This is the `.expect` file which is passing the test.

Reply:
Yes, have a look at how the test file for an explanation. The xUnit tests should have caught this, which I'll look into.

Reply:
https://github.com/dafny-lang/dafny/pull/2090

Reply:
Thanks, It was actually not correct, IDE removed extra space during formatting and that code was committed.